src/content/docs/en/guides/dev-toolbar.mdx

```diff
@@ -34,7 +34,14 @@ The dev toolbar aims to provide a quick and easy way to catch common issues duri
 
 ### Settings
 
-The Settings app allows you to toggle different settings for the development toolbar, such as verbose logging, and the ability to disable notifications.
+The Settings app allows you to tweak different settings for the development toolbar. For example:
```

Comment:
Agreed! Since the UI is self-explanatory, another advantage I can think of is that we don't have to update the docs if more options are added in the future. Thanks for this nice suggestion!